docroot/themes/humsci/humsci_basic/src/scss/elements/_details.scss

```diff
@@ -0,0 +1,90 @@
+details {
+  margin: hb-calculate-rems(16px) 0;
+}
+
+summary {
+  padding: hb-calculate-rems(16px);
+  position: relative;
+  font-weight: hb-theme-font-weight('semibold');
+  cursor: pointer;
+
+  @include hb-colorful {
+    border: $hb-thin-border;
+    @include hb-pairing-color('border-color', 'secondary');
+    @include hb-pairing-color('background-color', 'secondary');
+    @include hb-global-color('color', 'white');
+  }
+
+  // This is to position everything correctly in browsers that do not support the
+  // -webkit-details-marker pseudo element
+  display: flex;
+  align-items: center;
+
+  // Removes the default arrow icon
+  &::-webkit-details-marker{
+    display: block;
+    height: 0;
+    width: 0;
+    overflow: hidden;
+  }
+
+  &::after {
+    content: '';
+    display: block;
+    height: hb-calculate-rems(20px);
+    width: hb-calculate-rems(20px);
+
+    position: absolute;
+    top: hb-calculate-rems(18px);
+    right: hb-calculate-rems(18px);
+
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+
+    @include hb-colorful {
+      @include hb-menu-plus-icon('white');
+    }
+
+    details[open] & {
+      @include hb-colorful {
+        @include hb-menu-minus-icon('white');
+      }
+    }
+  }
+
+  &:hover,
+  details[open] & {
+    @include hb-colorful {
+      @include hb-pairing-color('background-color', 'secondary-darken-12');
+      @include hb-pairing-color('border-color', 'secondary-darken-12');
+    }
+  }
+
+  &:focus,
+  details[open] &:focus {
+    @include hb-colorful {
+      outline: none;
+      @include hb-global-color('border-color', 'white');
+      box-shadow: 0 0 0 hb-calculate-rems(2px) hb-get-pairing-color('secondary-active', $hb-colorful-default, $hc-colorful-pairings);
+      box-shadow: 0 0 0 hb-calculate-rems(2px) var(--palette--secondary-active);
+    }
+  }
+
+  a {
```

Comment:
sorry, could you include a comment that this is for IE 11? but this looks awesome now!

<img width="985" alt="Screen Shot 2020-04-14 at 10 04 34 AM" src="https://user-images.githubusercontent.com/12848168/79234052-aed95080-7e37-11ea-8454-b742b737f6b3.png">